e2e/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys
+import os
+
+sys.path.append(os.path.join(__file__))
```

Comment:
Correct, I put it in there to start with and then forgot to move it, thanks for the catch.